OpenDreamClient/Interface/Controls/ControlBrowser.cs

```diff
@@ -78,6 +79,23 @@ public ControlBrowser(ControlDescriptor controlDescriptor, ControlWindow window)
             if (newUri.Scheme == "byond" || (newUri.AbsolutePath == oldUri.AbsolutePath && newUri.Query != String.Empty)) {
                 context.DoCancel();
 
+                if (newUri.Host == "winset") { // Embedded winset. Ex: usr << browse("<a href=\"byond://winset?command=.quit\">Quit</a>", "window=quitbutton")
+                    // Strip the question mark out before parsing
+                    var queryParams = HttpUtility.ParseQueryString(newUri.Query.Substring(1));
+
+                    // We need to extract the control element (if one was included)
+                    string? element = queryParams.Get("element");
+                    queryParams.Remove("element");
+
+                    // Reassemble the query params without element then convert to winset syntax
+                    var query = queryParams.ToString();
+                    query = query!.Replace('&', ';');
```

Comment:
Could this cause issues with values that have `&` in them? Potentially something like `menu.name="&File"`

Reply:
```suggestion
                    query = query!.Replace('&', ';'); // TODO: More robust parsing
```